.circleci/create_circleci_config.py

```diff
@@ -447,6 +447,31 @@ def create_circleci_config(folder=None):
     if len(test_list) > 0:
         jobs.extend(REGULAR_TESTS)
 
+        # Extend the test files for cross test jobs
+        for job in jobs:
+            if job.job_name in ["tests_torch_and_tf", "tests_torch_and_flax"]:
+                extended_tests_to_run = set(test_list.split())
+                for test_path in copy.copy(extended_tests_to_run):
+                    dir_path, fn = os.path.split(test_path)
+                    if fn.startswith("test_modeling_tf_"):
+                        fn = fn.replace("test_modeling_tf_", "test_modeling_")
+                    elif fn.startswith("test_modeling_flax_"):
+                        fn = fn.replace("test_modeling_flax_", "test_modeling_")
+                    else:
+                        if job.job_name == "test_torch_and_tf":
+                            fn = fn.replace("test_modeling_", "test_modeling_tf_")
+                        elif job.job_name == "test_torch_and_flax":
+                            fn = fn.replace("test_modeling_", "test_modeling_flax_")
+                    new_test_file = str(os.path.join(dir_path, fn))
+                    if os.path.isfile(new_test_file):
+                        if new_test_file not in extended_tests_to_run:
+                            extended_tests_to_run.add(new_test_file)
+                job.tests_to_run = sorted(extended_tests_to_run)
+                fn = f'{job.job_name.replace("tests_", "filtered_test_list_")}.txt'
+                f_path = os.path.join(folder, fn)
+                with open(f_path, "w") as fp:
+                    fp.write(" ".join(job.tests_to_run))
```

Comment:
You can append to `f_path` in your for loop with a title being the job name so that everything ends up in the same file.